library/psa_crypto_driver_wrappers.c

```diff
@@ -64,6 +64,206 @@
 #endif
 
 /* Start delegation functions */
+psa_status_t psa_driver_wrapper_sign_message(
+    const psa_key_attributes_t *attributes,
+    const uint8_t *key_buffer,
+    size_t key_buffer_size,
+    psa_algorithm_t alg,
+    const uint8_t *input,
+    size_t input_length,
+    uint8_t *signature,
+    size_t signature_size,
+    size_t *signature_length )
+{
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+
+    /* Try dynamically-registered SE interface first */
+#if defined(MBEDTLS_PSA_CRYPTO_SE_C)
```

Comment:
If you make `psa_sign_message_internal` call `psa_driver_wrapper_sign_hash` instead, you can remove this block for the old-style SE interface. It's basically just unnecessary duplication like it is set up now.